src/usePost.js

```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+
+const baseURL = 'http://localhost:3000/';
+
+export const usePost = (url, title, body) => {
+    const [success, setSuccess] = useState(false);
+    const [loading, setLoading] = useState(false);
+    const [error, setError] = useState('');
+   
+    const postData = async () => {
```

Comment:
It would make it a little easier if you passed the params directly to the `postData` function so you don't have to pass them to `usePost`

```
const postData = async (title, body) => {
```